### ISSUE TITLE
mailto: links break the generated HTML

### ISSUE THREAD

**Issue by fiznool** [millermedeiros/gh-markdown-cli#8]
I am trying to do this:

``` bash
$ echo "[Send email](mailto:user@example.com)" | mdown
```

I am expecting this output:

``` html
<p><a href="mailto:user@example.com">Send email</a></p>
```

But instead I get this:

``` html
<p><a href="mailto:<a href='mailto:user@example.com'>user@example.com</a>">Send email</a></p>
```

I guess this might be a problem with the underlying parser? Or maybe this syntax isn't supported?

**Comment by millermedeiros** [comment]
It's an issue on the [markdown parser](https://github.com/isaacs/github-flavored-markdown).

If you do `echo "foo@bar.com" | mdown` it generates the "mailto" link `<p><a href="mailto:foo@bar.com">foo@bar.com</a></p>`.

Maybe we should change the parser, see #4.

**Comment by fiznool** [comment]
Yes, I can see the offending line: https://github.com/isaacs/github-flavored-markdown/blob/master/scripts/showdown.js#L193

Perhaps it would be easier to submit a patch to the markdown parser project to only add the anchor tag if the `mailto` is not already present, much in the same way that regular URL links are parsed: https://github.com/isaacs/github-flavored-markdown/blob/master/scripts/showdown.js#L188

**Comment by millermedeiros** [comment]
@fiznool based on the README on isaacs repository (that github abandoned the JS implementation) I guess we should change the markdown parser to one that follows GFM closer. I will leave this bug opened to remind we should replace the parser.

**Comment by Midnighter** [comment]
Just stumbled over this again :wink:
